examples/job/ml-train-model-predict-properties.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+# > <span style="color: red">**NOTE**</span>: This example utilizes features that are currently under maintenance, and may not work fully as-intended. For your convenience, please refer to the cached content of each calculation cell.
```

Comment:
as-intended doesn't need to be hyphenated.

Reply:
Noted. As this is a temporary notice anyway, let's proceed forward.